### PR TITLE
Made fossilise button attached to an IEntity to check if it is still alive

### LINKED ISSUE #3967: ObjectDisposedException when despawning all entities while paused
```
------------ Begin of Unhandled Exception Log ------------
The following exception prevented the game from running:

System.ObjectDisposedException: Cannot access a disposed object.
Object name: 'Microbe'.
  at Godot.Object.GetPtr (Godot.Object instance) [0x0001b] in /root/godot/modules/mono/glue/GodotSharp/GodotSharp/Core/Object.base.cs:43 
  at Godot.Spatial.GetGlobalTransform () [0x00001] in /root/godot/modules/mono/glue/GodotSharp/GodotSharp/Generated/GodotObjects/Spatial.cs:362 
  at Godot.Spatial.get_GlobalTransform () [0x00001] in /root/godot/modules/mono/glue/GodotSharp/GodotSharp/Generated/GodotObjects/Spatial.cs:128 
  at FossilisationButton.UpdatePosition () [0x0002c] in /home/ajax/source/Thrive/src/thriveopedia/fossilisation/FossilisationButton.cs:56 
  at StageHUDBase`1[TStage].UpdateFossilisationButtons () [0x00036] in /home/ajax/source/Thrive/src/general/StageHUDBase.cs:1455 
  at StageHUDBase`1[TStage]._Process (System.Single delta) [0x00076] in /home/ajax/source/Thrive/src/general/StageHUDBase.cs:534 
  at MicrobeHUD._Process (System.Single delta) [0x00001] in /home/ajax/source/Thrive/src/microbe_stage/MicrobeHUD.cs:82 

Please provide us with this log, thank you.
------------  End of Unhandled Exception Log  ------------
```

### PR DESCRIPTION
**Pull request by 84634E1A607A**
**Brief Description of What This PR Does**

This PR does some stuff...

Fixes #3967 

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [ ] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: src/thriveopedia/fossilisation/FossilisationButton.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@ public class FossilisationButton : TextureButton
     /// <summary>
     ///   The organism this button is attached to.
```
Comment (hhyyrylainen) [inline src/thriveopedia/fossilisation/FossilisationButton.cs:12]:
```suggestion
    ///   The entity (organism) this button is attached to.
```

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]
I think the approach is fine, but maybe @Oliveriver can comment as well? I added one small comment about changing one comment based on renaming one property.

**Review by hhyyrylainen — APPROVED** [review]
I didn't test, but the code looks now fine to me.

**Review by Oliveriver — APPROVED** [review]
Looks good and everything seems to work. Nice!

**Pull request merged**